### ISSUE TITLE
About the released pretrained model

### ISSUE THREAD

**Issue by suzhenghang** [facebookresearch/swav#11]
I wonder whether the released pretrained models were trained on uncurated data(1 billion random public non-EU images from Instagram) ?

**Comment by mathildecaron31** [comment]
All released models were pre-trained only on train set of ImageNet without labels.

**Comment by suzhenghang** [comment]
do you have a plan to release the pretrained models were trained on uncurated data?

**Comment by mathildecaron31** [comment]
Unfortunately, I don't think we can do that because of IP reasons. Let me ask around and come back to you.

**Comment by mathildecaron31** [comment]
It would indeed be a very long and painful process to release these models. For this reason, I am sorry to confirm that we are not planning to release our model trained on uncurated data unfortunately.
